### PR TITLE
add property  "extraParams"

### PR DESCRIPTION
**Pull request by taulinger**

### CODE REVIEW

File: test/modules/map/components/olMap/services/LayerService.test.js [hunk 1]
```diff
@@ -82,11 +82,12 @@ describe('LayerService', () => {
 				expect(wmsSource.getParams().VERSION).toBe('1.1.1');
 			});
 
-			it('converts a WmsGeoresource containing optional properties to a olLayer', () => {
+			fit('converts a WmsGeoresource containing optional properties to a olLayer', () => {
```
Comment (thiloSchlemmer) [inline test/modules/map/components/olMap/services/LayerService.test.js:85]: intended?

**Review by thiloSchlemmer — DISMISSED** [review]
Nice! One minor thing inline...

File: test/modules/map/components/olMap/services/LayerService.test.js (hunk 1 above)
Comment (taulinger) [reply test/modules/map/components/olMap/services/LayerService.test.js:85]: oops

**Review by thiloSchlemmer — DISMISSED** [review]

**Review by thiloSchlemmer — APPROVED** [review]
THX

**Pull request merged**